### ISSUE TITLE
Scan error on column index 2, name "checkpoint_write_time

### ISSUE THREAD

**Issue by copolycube** [prometheus-community/postgres_exporter#707]
**What did you do?**
started postgres-exporter
**What did you expect to see?**
no error
**What did you see instead? Under which circumstances?**
repeated errors 
```
postgres-exporter_1  | 2022-10-27T22:13:02.895786224Z ts=2022-10-27T22:13:02.895Z caller=collector.go:196 level=error msg="collector failed" name=bgwriter duration_seconds=0.010612212 err="sql: Scan error on column index 2, name \"checkpoint_write_time\": converting driver.Value type float64 (\"7.654199102e+09\") to a int: invalid syntax"
```
**Environment**
docker-compose environment
* System information:

	insert output of `uname -srm` here
Linux 5.15.0-52-generic x86_64

* postgres_exporter version:

	insert output of `postgres_exporter --version` here

```
c3f4e88ea08f                                                                      Up 2 days (healthy)   mainnet_postgres-exporter_1
```

* postgres_exporter flags:

```
insert list of flags used here
```

* PostgreSQL version:

	insert PostgreSQL version here

postgres:14-alpine

* Logs:
```
postgres-exporter_1  | 2022-10-27T22:04:28.674561053Z ts=2022-10-27T22:04:28.674Z caller=collector.go:196 level=error msg="collector failed" name=bgwriter duration_seconds=0.010603187 err="sql: Scan error on column index 2, name \"checkpoint_write_time\": converting driver.Value type float64 (\"7.65365925e+09\") to a int: invalid syntax"
postgres-exporter_1  | 2022-10-27T22:04:38.769158586Z ts=2022-10-27T22:04:38.769Z caller=collector.go:196 level=error msg="collector failed" name=bgwriter duration_seconds=0.010610871 err="sql: Scan error on column index 2, name \"checkpoint_write_time\": converting driver.Value type float64 (\"7.65365925e+09\") to a int: invalid syntax"
postgres-exporter_1  | 2022-10-27T22:04:48.854785223Z ts=2022-10-27T22:04:48.854Z caller=collector.go:196 level=error msg="collector failed" name=bgwriter duration_seconds=0.010743651 err="sql: Scan error on column index 2, name \"checkpoint_write_time\": converting driver.Value type float64 (\"7.65365925e+09\") to a int: invalid syntax"
postgres-exporter_1  | 2022-10-27T22:04:58.953229632Z ts=2022-10-27T22:04:58.953Z caller=collector.go:196 level=error msg="collector failed" name=bgwriter duration_seconds=0.010662268 err="sql: Scan error on column index 2, name \"checkpoint_write_time\": converting driver.Value type float64 (\"7.65365925e+09\") to a int: invalid syntax"
postgres-exporter_1  | 2022-10-27T22:05:09.055384564Z ts=2022-10-27T22:05:09.055Z caller=collector.go:196 level=error msg="collector failed" name=bgwriter duration_seconds=0.010670253 err="sql: Scan error on column index 2, name \"checkpoint_write_time\": converting driver.Value type float64 (\"7.65365925e+09\") to a int: invalid syntax"
postgres-exporter_1  | 2022-10-27T22:05:15.874119308Z ts=2022-10-27T22:05:15.873Z caller=collector.go:196 level=error msg="collector failed" name=bgwriter duration_seconds=0.010823771 err="sql: Scan error on column index 2, name \"checkpoint_write_time\": converting driver.Value type float64 (\"7.65365925e+09\") to a int: invalid syntax"
postgres-exporter_1  | 2022-10-27T22:05:19.143337857Z ts=2022-10-27T22:05:19.143Z caller=collector.go:196 level=error msg="collector failed" name=bgwriter duration_seconds=0.010649213 err="sql: Scan error on column index 2, name \"checkpoint_write_time\": converting driver.Value type float64 (\"7.65365925e+09\") to a int: invalid syntax"
postgres-exporter_1  | 2022-10-27T22:05:29.243588788Z ts=2022-10-27T22:05:29.243Z caller=collector.go:196 level=error msg="collector failed" name=bgwriter duration_seconds=0.010648652 err="sql: Scan error on column index 2, name \"checkpoint_write_time\": converting driver.Value type float64 (\"7.65365925e+09\") to a int: invalid syntax"
postgres-exporter_1  | 2022-10-27T22:05:39.319286036Z ts=2022-10-27T22:05:39.319Z caller=collector.go:196 level=error msg="collector failed" name=bgwriter duration_seconds=0.005675049 err="sql: Scan error on column index 2, name \"checkpoint_write_time\": converting driver.Value type float64 (\"7.65365925e+09\") to a int: invalid syntax"
postgres-exporter_1  | 2022-10-27T22:05:49.426788589Z ts=2022-10-27T22:05:49.426Z caller=collector.go:196 level=error msg="collector failed" name=bgwriter duration_seconds=0.010623444 err="sql: Scan error on column index 2, name \"checkpoint_write_time\": converting driver.Value type float64 (\"7.65365925e+09\") to a int: invalid syntax"
```

**Comment by copolycube** [comment]
with docker image 
`docker.io/bitnami/postgres-exporter@sha256:eba56617404069a44f0310136c4b01efbceb9edeca1089ecded16f37312ad6b9: Pulling from bitnami/postgres-exporter`

**Comment by sysadmind** [comment]
This was fixed in #666 and released in postgres_exporter v0.11.1. The bitnami docker image is not official and I'm not very sure what version of the exporter that image ships with. Official images can be found here: https://quay.io/repository/prometheuscommunity/postgres-exporter?tab=tags&tag=latest

**Comment by copolycube** [comment]
I confirm that _not_ using the bitnami version helped solve this issue. Thanks !
